index.js

```diff
@@ -85,11 +92,11 @@ Block.prototype.isGenesis = function () {
  * @method setGenesisParams
  */
 Block.prototype.setGenesisParams = function () {
-  this.header.gasLimit = params.genesisGasLimit.v
-  this.header.difficulty = params.genesisDifficulty.v
-  this.header.extraData = params.genesisExtraData.v
-  this.header.nonce = params.genesisNonce.v
-  this.header.stateRoot = params.genesisStateRoot.v
+  this.header.gasLimit = this._common.genesis().gasLimit
+  this.header.difficulty = this._common.genesis().difficulty
+  this.header.extraData = this._common.genesis().extraData
+  this.header.nonce = this._common.genesis().nonce
+  this.header.stateRoot = this._common.genesis().stateRoot
   this.header.number = new Buffer([])
```

Comment:
@vpulim Side note, just realized this from another context: ``this.header.number`` should be ``0`` and not the emtpy buffer, shouldn't it? So ``Buffer.from([0])`` and not ``new Buffer([])``?

Reply:
Then I would update this as well with this PR.